packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -108,6 +108,32 @@ module.exports = {
       }
     ],
     loaders: [
+      // Default loader: load all assets that are not handled
+      // by other loaders with the url loader.
+      // Note: This list needs to be updated with every change of extensions
+      // the other loaders match.
+      // E.g., when adding a loader for a new supported file extension,
```

Comment:
I suppose the "negative" loader could be calculated from the others, at the price of making it imperative.
The least-invasive change might remove the loader and add a line at the end of the config something like:

`module.exports.loaders.push(fn(module.exports.loaders));`

Would you like something along these lines?